src/om_bootstrap/panel.cljs

```diff
@@ -16,21 +16,48 @@
    {(s/optional-key :on-select) (sm/=> s/Any s/Any)
     (s/optional-key :header) t/Renderable
     (s/optional-key :footer) t/Renderable
-    (s/optional-key :list-group) t/Renderable}))
+    (s/optional-key :list-group) t/Renderable
+    (s/optional-key :collapsed?) s/Bool}))
 
 (sm/defn panel :- t/Component
   [opts :- Panel & children]
   (let [[bs props] (t/separate Panel opts {:bs-class "panel"
                                            :bs-style "default"})
-        classes (assoc (t/bs-class-set bs)
-                  :panel true)]
+        classes (assoc (t/bs-class-set bs) :panel true)]
     (d/div (u/merge-props props {:class (d/class-set classes)})
            (when-let [header (:header bs)]
              (d/div {:class "panel-heading"}
                     (u/clone-with-props header {:class "panel-title"})))
            (when-not (= 0 (count (filter identity children)))
-             (d/div {:class "panel-body" :ref "body"} children))
+             (d/div {:class (str "panel-body" (when (:collapsed? bs) " collapse"))
+                     :ref "body"}
+                    children))
            (when-let [list-group (:list-group bs)]
              list-group)
-           (when-let [footer(:footer bs)]
+           (when-let [footer (:footer bs)]
              (d/div {:class "panel-footer"} footer)))))
+
+;; ## Collapsible Panel
+
+(defcomponentk collapsible-panel*
+  "Generates a collapsible panel component resposible for its own toggled state.
+   The :collapsed? state is handled through a collapsible mixin."
+  [owner state]
+  (:mixins m/collapsible-mixin)
+  (render [_]
+    (let [{:keys [opts children]} (om/get-props owner)
+          is-collapsed? ((aget owner "isPanelCollapsed") owner)
+          toggle! (fn [_] ((aget owner "toggleCollapsed") owner) false)
```

Comment:
I added the "false" to prevent jumping to the top of the page. I think this behaves like (.preventDefault e). Should I leave this in?